cmd/rpcfuzz/rpcfuzz.go

```diff
@@ -1879,9 +1898,14 @@ func init() {
 	testFuzz = flagSet.Bool("fuzz", false, "Flag to indicate whether to fuzz input or not.")
 	testFuzzNum = flagSet.Int("fuzzn", 100, "Number of times to run the fuzzer per test.")
 	seed = flagSet.Int64("seed", 123456, "A seed for generating random values within the fuzzer")
+	testOutputExportPath = flagSet.String("export-path", "", "The directory export path of the output of the tests. Must pair this with either --json, --csv, --md, or --html")
```

Comment:
Just a question, is there a default output mode or the command will fail if the user forgets to specify one of the format flags (csv, html, etc.)?

Reply:
good question, for now I've just 1) inform the user on the flag usage level and 2) added a warn message if the export path is present without an export type